DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -162,13 +162,31 @@ extension MultiplicationExpression_Tests {
 // MARK: - CustomStringConvertible
 extension MultiplicationExpression_Tests {
     
-    func test_CustomStringConvertible() {
+    func test_CustomStringConvertible_constantAndDie() {
         let expression = 2 * d(10)
-        let expected = "2d10"
+        let expected = "\(String(c(2)))\(String(d(10)))"
```

Comment:
Nope, just missed this. I had them all as `String()` before, then went and removed them. I will remove this one.